### PR TITLE
use ctrl + enter to submit comment forms

### PR DESCRIPTION
**Pull request by motatoes**
addresses #727

### CODE REVIEW

File: resources/dmmd-preview.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+        var $form = $preview.parents('form').first();
         var $update = $preview.find('.dmmd-preview-update');
         var $content = $preview.find('.dmmd-preview-content');
         var preview_url = $preview.attr('data-preview-url');
         var $textarea = $('#' + $preview.attr('data-textarea-id'));
 
+        // submit the form if ctrl + enter is pressed in pagedown textarea
+        $textarea.keydown(function (ev) {
+            // Ctrl-Enter pressed
+            if (ev.ctrlKey && ev.keyCode == 13) {
```
Comment (Xyene) [inline resources/dmmd-preview.js]: This doesn't work, at least on my Chrome install. Pressing Ctrl+Enter into the textbox at https://api.jquery.com/keypress/ has the `keyCode` be 10, and just Enter be 13. `.keyCode` isn't very cross-browser-compatible; you should be using the jQuery-provided `.which` instead.

File: resources/dmmd-preview.js [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+                $form.submit()
```
Comment (Xyene) [inline resources/dmmd-preview.js]: Semicolon at end.

File: resources/dmmd-preview.js [hunk 3]
```diff
@@ -1,10 +1,20 @@
 $(function () {
     window.register_dmmd_preview = function ($preview) {
+
+        var $form = $preview.parents('form').first();
         var $update = $preview.find('.dmmd-preview-update');
         var $content = $preview.find('.dmmd-preview-content');
         var preview_url = $preview.attr('data-preview-url');
         var $textarea = $('#' + $preview.attr('data-textarea-id'));
 
+        // submit the form if ctrl + enter is pressed in pagedown textarea
```
Comment (Xyene) [inline resources/dmmd-preview.js]: Start with uppercase letter; end with period.

File: resources/dmmd-preview.js [hunk 4]
```diff
@@ -1,10 +1,20 @@
...
+        $textarea.keydown(function (ev) {
+            // Ctrl-Enter pressed
```
Comment (Xyene) [inline resources/dmmd-preview.js]: Be consistent in comments, used "ctrl + enter" above and "Ctrl-Enter" here, "Ctrl+Enter" everywhere sounds good.

File: resources/dmmd-preview.js [hunk 5]
```diff
@@ -1,10 +1,20 @@
...
+            if (ev.ctrlKey && ev.keyCode == 13) {
+                $form.submit()
+            }
+        })
```
Comment (Xyene) [inline resources/dmmd-preview.js]: Semicolon at end.

**Comment by motatoes** [comment]
@Xyene I addressed the changes in the previous two commits

File: resources/dmmd-preview.js [hunk 6]
```diff
@@ -1,10 +1,20 @@
 $(function () {
     window.register_dmmd_preview = function ($preview) {
+
```
Comment (quantum5) [inline resources/dmmd-preview.js]: Extra newline

**Comment by Xyene** [comment]
I would be happy to merge this since I think it works, but there's still a (minor) unaddressed comment: https://github.com/DMOJ/site/pull/737/files#r223161009

**Comment by motatoes** [comment]
Done :D

**Pull request merged**